uygarkov/diffusion2d.py

```diff
@@ -0,0 +1,71 @@
+"""
+Solving the two-dimensional diffusion equation
+Example acquired from https://scipython.com/book/chapter-7-matplotlib/examples/the-two-dimensional-diffusion-equation/
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from .output import output_plots, create_plot
+
+def solve(dx,dy,D):
```

Comment:
```suggestion
def solve(dx = 0.1,dy = 0.1 ,D = 1):
```
Giving default values to the input arguments of a function that the user will call is important.